packages/orchestration/src/orchestration-types.ts

```diff
@@ -33,17 +33,16 @@ export type LlmConfig = LLMModuleConfig & {
  * Orchestration module configuration.
  */
 export interface OrchestrationModuleConfig {
-  // TODO: remove "config" for all the configs below
```

Comment:
Leftover comment

Reply:
you are looking at the left side, but it is gone on the right ;)